lgsm/functions/check_permissions.sh

```diff
@@ -149,17 +163,11 @@ fn_sys_perm_errors_detect(){
 	sysdirpermerror="0"
 	classdirpermerror="0"
 	netdirpermerror="0"
-	# Check permissions/
-	# /sys, /sys/class and /sys/class/net should be readable & executable.
+	# Check permissions
+	# /sys, /sys/class and /sys/class/net should be readable & executable
 	if [ ! -r "/sys" ]||[ ! -x "/sys" ]; then
```

Comment:
FreeBSD does not have `/sys`. I had a check here for this. It looks like it got lost in the latest merge commit.